### PR TITLE
Costsms kbr

### PR DESCRIPTION
**Pull request by kbr-odoo**

### CODE REVIEW

File: sms_marketing/SMS_mkt_build_send-and_get_reports_from_a_SMS_mailing.rst [hunk 1]
```diff
@@ -21,57 +20,78 @@ Go to :menuselection:`SMS Marketing --> Create`.
 .. image:: ./media/sms_marketing1.png
    :align: center
 
-First, I’ll be able to specify a *subject* name that will help me to remember what the mailing is about.
-Under *recipients*, I’ll choose to whom I’d like the SMS to be sent. If I choose *Contact*, all my Odoo contacts (vendor, customers, etc.) will receive my SMS.
-From there, I can refine my recipient list with the *add filter* option. In the example below, I choose to send it to my contacts living in Belgium.
+First, you will be able to specify a *subject* name that will help you to remember what the mailing
+is about.
+Under *recipients*, choose to whom you would like the SMS to be sent. If you choose *Contact*, all your
```
Comment (AntoineVDV) [inline sms_marketing/SMS_mkt_build_send-and_get_reports_from_a_SMS_mailing.rst]: Line break not rendered in html

File: sms_marketing/SMS_mkt_build_send-and_get_reports_from_a_SMS_mailing.rst [hunk 2]
```diff
@@ -21,57 +20,78 @@ Go to :menuselection:`SMS Marketing --> Create`.
...
+Odoo contacts (vendor, customers, etc.) will receive your SMS.
+From there, you can refine your recipient list with the *add filter* option. In the example below,
```
Comment (AntoineVDV) [inline sms_marketing/SMS_mkt_build_send-and_get_reports_from_a_SMS_mailing.rst]: Same as above.

File: sms_marketing/SMS_mkt_contact_lists_and_blacklist.rst [hunk 3]
```diff
@@ -1,34 +1,43 @@
...
+an opportunity, for example).
 
 .. image:: ./media/sms_marketing10.png
    :align: center
 
-This is important as I and my team can easily keep track and see a history of interactions with my contacts or prospects, granting me the ability to better understand them.
-Example: my sales representative can easily find out which SMS mailing promotions a customer has received or not.
+This is important as you and your team can easily keep track and see a history of interactions with
+your contacts or prospects, granting you the ability to better understand them.
+Example: your sales representative can easily find out which SMS mailing promotions a customer has
```
Comment (AntoineVDV) [inline sms_marketing/SMS_mkt_contact_lists_and_blacklist.rst]: Same as above.

File: sms_marketing/SMS_mkt_contact_lists_and_blacklist.rst [hunk 4]
```diff
@@ -1,34 +1,43 @@
...
 
 
 Blacklist
-==================================
-If I’m coming from another software and have a list of clients who have already asked to be blacklisted from my news, I can import those entries in my database.
-For that, I’ll go to :menuselection:`Configuration --> Phone Blacklist --> Import`.
+=========
+If you are coming from another software and have a list of clients who have already asked to be
+blacklisted, you can import those entries in your database.
+For that, go to :menuselection:`Configuration --> Phone Blacklist --> Import`.
```
Comment (AntoineVDV) [inline sms_marketing/SMS_mkt_contact_lists_and_blacklist.rst]: Same as above.

File: sms_marketing/SMS_mkt_integrations_and_template.rst [hunk 5]
```diff
@@ -1,58 +1,63 @@
...
 .. image:: ./media/sms_marketing14.png
    :align: center
 
 Sending SMSs through the Contacts app
 ======================================
-Sending SMSs through the contact’s form is available by default in Odoo. It makes my work easier if I need to send an SMS to a specific contact, for example.
-For that, I’ll go to the *Contacts* app, select the contact and click on the *SMS* icon next to the phone number.
+Sending SMSs through the contact’s form is available by default in Odoo. It makes your work easier
+if you need to send an SMS to a specific contact, for example.
+For that, go to the *Contacts* app, select the contact and click on the *SMS* icon next to the
```
Comment (AntoineVDV) [inline sms_marketing/SMS_mkt_integrations_and_template.rst]: Same as above.

File: sms_marketing/SMS_mkt_integrations_and_template.rst [hunk 6]
```diff
@@ -64,5 +69,7 @@ SMS Templates` and create the content I need.
    - *SEPA debit payment provider*: send a verification code to your customers.
    - *Studio*: send an SMS according to your needs by using automated actions.
 
-For more information about SMS integrations in Odoo and a list of frequently asked questions, go to `Pricing and FQA <fqapricing.html>`__.
+.. tip::
+   For more information about SMS integrations in Odoo and a list of frequently asked questions,
+   go to `Pricing and FQA <faqpricing.html>`__.
```
Comment (AntoineVDV) [inline sms_marketing/SMS_mkt_integrations_and_template.rst]:
```suggestion
   go to :doc:`faqpricing`.
```

**Comment by AntoineVDV** [comment]
Pretty good overall :+1: 
Pay attention to the special behavior of line breaks in RST and to hyperlinks and there's a good chance that the next PR will be merged in one passe ;)

File: sms_marketing/SMS_mkt_build_send-and_get_reports_from_a_SMS_mailing.rst [hunk 7]
```diff
@@ -21,57 +20,81 @@ Go to :menuselection:`SMS Marketing --> Create`.
 .. image:: ./media/sms_marketing1.png
    :align: center
 
-First, I’ll be able to specify a *subject* name that will help me to remember what the mailing is about.
-Under *recipients*, I’ll choose to whom I’d like the SMS to be sent. If I choose *Contact*, all my Odoo contacts (vendor, customers, etc.) will receive my SMS.
-From there, I can refine my recipient list with the *add filter* option. In the example below, I choose to send it to my contacts living in Belgium.
+| First, you will be able to specify a *subject* name that will help you to remember what the
+  mailing is about.
+
```
Comment (AntoineVDV) [inline sms_marketing/SMS_mkt_build_send-and_get_reports_from_a_SMS_mailing.rst]: You should either:
- use blank lines in RST to create a new paragraph in HTML
or
- use the `|` notation to force a line break within a paragraph

but not both at the same time.

File: sms_marketing/SMS_mkt_build_send-and_get_reports_from_a_SMS_mailing.rst [hunk 8]
```diff
@@ -21,57 +20,81 @@ Go to :menuselection:`SMS Marketing --> Create`.
...
+| Under *recipients*, choose to whom you would like the SMS to be sent. If you choose *Contact*,
+  all your Odoo contacts (vendor, customers, etc.) will receive your SMS.
+
```
Comment (AntoineVDV) [inline sms_marketing/SMS_mkt_build_send-and_get_reports_from_a_SMS_mailing.rst]: Same as above.

File: sms_marketing/SMS_mkt_contact_lists_and_blacklist.rst [hunk 9]
```diff
@@ -1,34 +1,44 @@
...
-I’ll choose a name for my *Mailing list*, and activate the option *Public* if I would like to make the mailing list accessible to my recipients in the unsubscription page to allow them to update their subscription preferences.
+Contact lists
+=============
+| Go to :menuselection:`Contacts lists --> Contacts lists --> Create`.
+| Choose a name for your *Mailing list*, and activate the option *Public* if you would like to make
+  the mailing list accessible to your recipients in the unsubscription page to allow them to update
+  their subscription preferences.
 
-Once inside the list, I can again *Create* or *Import* contacts for that specific list.
+| Once inside the list, you can again *Create* or *Import* contacts for that specific list.
```
Comment (AntoineVDV) [inline sms_marketing/SMS_mkt_contact_lists_and_blacklist.rst]: Same as above.

File: sms_marketing/SMS_mkt_contact_lists_and_blacklist.rst [hunk 10]
```diff
@@ -1,34 +1,44 @@
...
 
 .. image:: ./media/sms_marketing10.png
    :align: center
+   :height: 300
 
-This is important as I and my team can easily keep track and see a history of interactions with my contacts or prospects, granting me the ability to better understand them.
-Example: my sales representative can easily find out which SMS mailing promotions a customer has received or not.
+| This is important as you and your team can easily keep track and see a history of interactions
+  with your contacts or prospects, granting you the ability to better understand them.
+| Example: your sales representative can easily find out which SMS mailing promotions a customer has
```
Comment (AntoineVDV) [inline sms_marketing/SMS_mkt_contact_lists_and_blacklist.rst]: Good, that's how you do it :+1:

**Comment by kbr-odoo** [comment]
> Did you compress `sms_marketing16.png` ? It just had a 1400% increase in weight :D

No. I replaced it because otherwise, the image was unreadable.

**Comment by AntoineVDV** [comment]
I compressed it and made some other little (technical) changes. You don't need to do anything (unless you want to bring some more changes).

**Comment by kbr-odoo** [comment]
Well, as I said, I replaced it with the original one and did not compress that time because otherwise, the image is unreadable, and that how it is now, and I did not want that. But, ok, I guess. Thank you for merging it.

**Comment by AntoineVDV** [comment]
> Well, as I said, I replaced it with the original one and did not compress that time because otherwise, the image is unreadable, and that how it is now, and I did not want that. But, ok, I guess. Thank you for merging it.

Resolved

**Comment by kbr-odoo** [comment]
These docs were reviewed not long ago. The only thing I did was to change pronouns.

**Pull request merged**